src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/CallInstruction.cs

```diff
@@ -49,9 +49,9 @@ public static CallInstruction Create(MethodInfo info, ParameterInfo[] parameters
                 return GetArrayAccessor(info, argumentCount);
             }
 
+#if !FEATURE_DLG_INVOKE
             return new MethodInfoCallInstruction(info, argumentCount);
-#if FEATURE_DLG_INVOKE
```

Comment:
What is FEATURE_DLG_INVOKE?  Do we ever compile with it?  I'm wondering why this being dead code wasn't previously triggering any failures.